packages/orbit-components/src/Alert/index.tsx

```diff
@@ -74,7 +74,7 @@ const ContentWrapper = ({
   return (
     <div
       className={cx(
-        "flex min-h-[20px] items-center",
+        "min-h-icon-medium flex items-center",
```

Comment:
AFAIK this change was made automatically during the committing phase. It could be manually excluded, but this "typo" change doesn't affect the final result.

Reply:
Nope, unexpected. The value might be the same but this `ContentWrapper` is not an icon per se. I will revert.